README.Rmd

```diff
@@ -22,13 +22,156 @@ knitr::opts_chunk$set(
+
+In addition to filtering and log2 transformation it is also advised to median normalise your data to equal out small differences in overall sample intensities that result from unequal sample concentrations. protti provides the `median_normalisation` function for this purpose. This function generates an additional column called `normalised_intensity_log2` that contains the normalised intensities.
+
+```{r normalisation, message=FALSE, warning=FALSE}
+normalised_data <- data %>% 
+  median_normalisation(sample = sample,
+                       intensity_log2 = peptide_intensity_missing)
+```
+
+#### Assign Missingness
+
+The next step is to deal with missing data points. You could chose to impute missing data in a later step, but this is only recommended if not a large proportion of your data is missing and if you know what you are doing. In order to calculate statistical significance of differentially abundant peptides or proteins we would like to have at least a minimum number of observations per condition. The protti function `assign_missingness` checks for each pair of treatment to reference condition if this number is satisfied and assigns a missingness type to each comparison. Comparisons that have too few observations are labeled `NA`. In order to be labeled as `NA` by default both conditions need to have less than 0.7 times the number of maximum observations adjusted downward. If you performed an experiment with 4 replicates that means that both conditions need to contain at least 2 observations. You can read the exact details in the documentation of this function and also adjust the thresholds if you want to be more or less conservative with how many datapoints to retain. 
+
+```{r assign_missingness, message=FALSE, warning=FALSE}
+data_missing <- normalised_data %>% 
+  assign_missingness(sample = sample,
+                     condition = condition,
+                     grouping = peptide,
+                     intensity = normalised_intensity_log2,
+                     ref_condition = "condition_1",
+                     retain_columns = c(protein, change_peptide))
+
+# Next to the columns it generates, assign_missingness only contains the columns you provide as input in its output. 
+# If you want to retain additional columns you can provide them in the retain_columns argument.
+```
+
+Note: Instead of "peptide" in the `grouping` argument you can provide protein IDs in case you are working with protein abundance data. However, then intensities should also contain protein intensities and not peptide intensities.
+
+#### Calculate Differential Abundance and Significance
+
+For the calculation of abundance changes and the associated significance protti provides the function `diff_abundance`. You can choose between different statistical methods. For this example we will chose a Welch's t-test. 
```

Comment:
*choose

Reply:
You can also mention that here you obtain p-values and p-values that were adjusted for multiple testing (q-values, using BH) and that it is advisable to use q-values.